igor-web/src/main/java/com/netflix/spinnaker/igor/artifactory/model/ArtifactoryItem.java

```diff
@@ -32,9 +32,9 @@
   private List<ArtifactoryArtifact> artifacts;
 
   @Nullable
-  public Artifact toMatchableArtifact(ArtifactoryRepositoryType repoType, String baseUrl) {
-    switch (repoType) {
-      case Maven:
+  public Artifact toMatchableArtifact(ArtifactoryRepositoryType repositoryType, String baseUrl) {
+    switch (repositoryType.getRepoTypeString()) {
+      case "Maven":
```

Comment:
Not sure why you're converting these to a string. By doing so, you'll remove IntelliJ's (and other IDEs) ability to know if the switch statement is exhaustive or not. I'd recommend changing this back to switching on the enum type.

Reply:
Agree, reverted change.